internal/api/v1beta1/group.go

```diff
@@ -38,6 +39,7 @@ type GroupService interface {
 
 var (
 	grpcGroupNotFoundErr = status.Errorf(codes.NotFound, "group doesn't exist")
+	grpcMinOwnerCounrErr = status.Errorf(codes.InvalidArgument, "group must have at least one owner. Please add another owner before removing this one")
```

Comment:
we can rephrase this as `group must have at least one owner, consider adding another before removing`